staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go

```diff
@@ -146,3 +150,33 @@ type CustomResourceValidation struct {
 	// OpenAPIV3Schema is the OpenAPI v3 schema to be validated against.
 	OpenAPIV3Schema *JSONSchemaProps
 }
+
+// CustomResourceSubResources defines the status and scale subresources for CustomResources.
+type CustomResourceSubResources struct {
+	// Status denotes the status subresource for CustomResources
+	Status *CustomResourceSubResourceStatus
+	// Scale denotes the scale subresource for CustomResources
+	Scale *CustomResourceSubResourceScale
+}
+
+// CustomResourceSubResourceStatus defines how to serve the HTTP path <CR Name>/status.
+// status is represented by the `.status` JSON path.
+type CustomResourceSubResourceStatus struct{}
+
+// CustomResourceSubResourceScale defines how to serve the HTTP path <CR name>/scale.
+type CustomResourceSubResourceScale struct {
+	// required, e.g. ".spec.replicas".
```

Comment:
nit: first a sentence what it does: Defines the JSON path inside of a CR that corresponds to Scale.Spec.Replicas.

Then meta info about optional + destrictions etc.

Reply:
still open

Reply:
The external types have it. Just copy the Godoc.